Element84/cmr-stac-api-proxy
# Feature/staccmr 18 express

Migrated application to use express and aws-serverless-express. This move was to simplify the request/response mechanism and introduce a common syntax to creating endpoints.

To test:
- navigate to `/` in the application
- ensure that the redirect happens to docs
- ensure that docs/index.html shows
